src/menus/optionsMenu.cpp

```diff
@@ -169,6 +169,38 @@ OptionsMenu::OptionsMenu()
         destroy();
     }))->setSize(GuiElement::GuiSizeMax, 50);
 
+    //Select the language
+    (new GuiLabel(interface_page, "LANGUAGE_OPTIONS_LABEL", tr("Language (applies on back)"), 30))->addBackground()->setSize(GuiElement::GuiSizeMax, 50);
+    
+    //Don't wan't scripts/locale. Also, compatible with android assets.
+    DirectoryResourceProvider* lang_prov {new DirectoryResourceProvider("resources/locale")};
+    std::vector<string> languages {lang_prov->findResources("*.po")};
```

Comment:
Why not `findResources("locale/*po")`? that saves constructing the resourceprovider.

Reply:
This would not search and return scripts/locale/*po scenario specific translations as well ?

Reply:
I don't think it's better to list more files to filter out after... But another possibility would be to have the global langage files in another directory ? So it's possible to search them with another hierarchy.

Reply:
But you would already filter those out at line 186?